src/descriptors/portfolio_structs.json

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
why we need autogeneration for this application?. This is more complicated that keeping simple types.

Reply:
Eventually we will need to build more detailed CEM models for GMLC and this should not complicate it too much.